crates/bevy_ui/src/accessibility.rs

```diff
@@ -0,0 +1,157 @@
+use bevy_a11y::{
+    accesskit::{NodeBuilder, Rect, Role},
+    AccessibilityNode,
+};
+use bevy_app::{App, Plugin};
+
+use bevy_ecs::{
+    prelude::Entity,
+    query::{Changed, Or, Without},
+    system::{Commands, Query},
+};
+use bevy_hierarchy::Children;
+
+use bevy_render::prelude::Camera;
+use bevy_text::Text;
+use bevy_transform::prelude::GlobalTransform;
+
+use crate::{
+    prelude::{Button, Label},
+    Node, UiImage,
+};
+
+fn calc_name(texts: &Query<&Text>, children: &Children) -> Option<Box<str>> {
+    let mut name = None;
+    for child in children.iter() {
+        if let Ok(text) = texts.get(*child) {
+            let values = text
+                .sections
+                .iter()
+                .map(|v| v.value.to_string())
+                .collect::<Vec<String>>();
+            name = Some(values.join(" "));
+        }
+    }
+    name.map(|v| v.into_boxed_str())
+}
+
+fn calc_bounds(
+    camera: Query<(&Camera, &GlobalTransform)>,
+    mut nodes: Query<
+        (&mut AccessibilityNode, &Node, &GlobalTransform),
+        Or<(Changed<Node>, Changed<GlobalTransform>)>,
+    >,
+) {
+    if let Ok((camera, camera_transform)) = camera.get_single() {
```

Comment:
Can you please either point to a multi-camera UI example, or let me know how to determine which camera to use when calculating the correct translation? I'm trying to avoid taking responsibility for more of bevy_ui's rough edges than I absolutely have to, and I think I'd rather this fail with outliers for now, than have to make these types of decisions on my own. But the examples I've worked with all seem to use a single camera, so...